tools/coco_limit_image_size.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import argparse
+import json
+import os
+from math import floor
+
+import cv2
+from tqdm import tqdm
+
+
+def parse_args():
+    args = argparse.ArgumentParser()
+    args.add_argument('annotation', help='Path to COCO annotation (*.json).')
+    args.add_argument('input_root', help='Path to input images root folder.')
+    args.add_argument('output_root', help='Path to output images root folder.')
+    args.add_argument('--limit_image_size', type=int, nargs=2, default=(1080, 1920),
+                      help='Resize images with size greater than specified here size (h, w)! '
+                           'keeping aspect ratio.')
+    return args.parse_args()
+
+
+def main():
+    args = parse_args()
+    assert args.input_root != args.output_root
+    assert not os.path.exists(args.output_root)
+
+    with open(args.annotation) as f:
+        content = json.load(f)
+
+    resized = 0
+
+    for i, image_info in enumerate(tqdm(content['images'])):
+        path = os.path.join(args.input_root, image_info['file_name'])
+        image = cv2.imread(path)
+        if image is None:
+            print(path)
```

Comment:
Could you please make this message more informative?

Reply:
fixed